src/app/header/header.component.ts

```diff
@@ -47,4 +48,10 @@ export class HeaderComponent implements OnInit {
   login(): void{
     this.store.dispatch(new Login());
   }
+  linksGenerating(){
+    let role = this.store.selectSnapshot<string>(UserRegistrationState.role);
+    let urlRole = (role==='organization') ? 'provider':'parent';
```

Comment:
you can move it to enum and use it as an associative array

Reply:
I didn't understand how or what to do;
I will do it later